src/pages/dashboardTab/store/dashboardReducer.js

```diff
@@ -617,9 +646,63 @@ const reducers = {
           filters: [],
         },
         widgets: getWidgetsInitData(item.data, widgetsData),
+        dataCaseSelected: {
+          selectedRowInfo: [],
+          selectedRowIndex: [],
+        },
+        dataSampleSelected: {
+          selectedRowInfo: [],
+          selectedRowIndex: [],
+        },
+        dataFileSelected: {
+          selectedRowInfo: [],
+          selectedRowIndex: [],
+        },
 
       } : { ...state };
   },
+  CLEAR_ALL_FILTER: (state, item) => {
+    const checkboxData = customCheckBox(item.data, facetSearchData, 'count');
+    fetchDataForDashboardTab(state.currentActiveTab, null, null, null);
+    return item.data
+      ? {
+        ...state.dashboard,
+        isFetched: true,
+        isLoading: false,
+        hasError: false,
+        error: '',
+        stats: getStatInit(item.data, statsCount),
+        allActiveFilters: allFilters(),
+        filteredSubjectIds: null,
+        filteredSampleIds: null,
+        filteredFileIds: null,
+        subjectOverView: {
+          data: item.data.subjectOverViewPaged,
+        },
+        checkboxForAll: {
+          data: checkboxData,
+        },
+        checkbox: {
+          data: checkboxData,
+        },
+        datatable: {
+          dataCase: item.data.subjectOverViewPaged,
+          dataSample: item.data.sampleOverview,
+          dataFile: item.data.fileOverview,
+          filters: [],
+        },
+        dataCaseSelected: {
+          ...state.dataCaseSelected,
+        },
+        dataSampleSelected: {
+          ...state.dataSampleSelected,
+        },
+        dataFileSelected: {
+          ...state.dataFileSelected,
+        },
+        widgets: getWidgetsInitData(item.data, widgetsData),
+      } : { ...state };
+  },
   CLEAR_ALL: (state, item) => {
```

Comment:
Can we keep only one CLEAR_ALL or CLEAR_ALL_FILTERS as they are duplicate reducers?